libraries/dotnet/EnergyOrigin.TokenValidation/version.yaml

```diff
@@ -1 +1 @@
-version: 1.0.8
+version: 1.0.9
```

Comment:
Why is this needed?

Reply:
Jakob's problem with the Github actions setup pipeline failing. he was doing testing on this branch.